14_Docker/yarn_test/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM centos:7
+MAINTAINER Maksim wardomenmax@gmail.com
+LABEL HDFS Hadoop
+
+RUN yum clean all && \
+	yum list updates && \  
+	yum -y install java-1.8.0-openjdk-devel
+RUN yum -y install wget
+RUN yum -y install openssh-server openssh-clients
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk
+ENV PATH $PATH:$JAVA_HOME/bin
+ENV HADOOP_HOME /usr/local/hadoop
+ENV HADOOP_CONF_DIR=$HADOOP_HOME/etc/hadoop
+
+ARG HOST_NAME=scylla
+ARG CLIENT_NAME=charybdis
+ARG HADOOP_VERSION=3.1.2
+#RUN mkdir downloads \
```

Comment:
It is advised to cleanup commented lines of code before creating Pull Request
Consider this as a note, not as a mistake

Reply:
Thanks. )